### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Focus order is illogical while we navigating the list items present under Repository in ACR Explorer

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1416]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. In "Azure Explorer" , select the pushed image under container registries and press shift+F10 to open the context menu.
4. Navigate to and select "Open ACR Explorer" menu item .
5. The open window , tab to list items which is present under Repository then select any list item after that observe the keyboard focus

**Expected Result:**
Keyboard focus should be in list item which is present under Repository

**Actual Result:**
Keyboard focus is moving to "Refresh" button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![acr explorer focus order issue](https://user-images.githubusercontent.com/22956130/35717110-2599c5cc-0803-11e8-975e-c426c8dc7bf3.jpg)

**Comment by jdneo** [comment]
Fixed

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.Issue got fixed.
Please add closed tag 
![1416-fixed](https://user-images.githubusercontent.com/22956130/39760201-59f2e896-52f2-11e8-8eb7-c78cedf6c02d.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749872
